components/basic_widgets/CheckBoxWidget.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div>
+    <div class="div__checkbox-background">
+      <b-form-checkbox-group
+        v-model="selected"
+        :options="checkbox_options"
+        stacked
+        @change="change_method"
+      ></b-form-checkbox-group>
+    </div>
+  </div>
+</template>
+<script>
+import Vue from "vue";
+import { BFormCheckboxGroup } from "bootstrap-vue";
+Vue.component("BFormCheckboxGroup", BFormCheckboxGroup);
+
+export default {
+  name: "CheckBoxWidget",
+  props: {
+    checkbox_options: {
+      type: Array,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      selected: [],
+    };
+  },
+  methods: {
+    change_method() {
+      this.$emit("checkbox-selected", this.selected);
+    },
+  },
+};
+</script>
+<style type="text/css">
+.div__checkbox-background {
+  transform: rotate(0deg);
+  box-sizing: border-box;
+  padding: 0px;
+  margin: 0px;
+  background: rgb(17, 17, 17);
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 140px;
+  height: 150px;
+  font-family: Muli;
+  font-weight: normal;
+  font-style: normal;
+  text-decoration: none;
+  font-size: 15px;
+  color: rgb(183, 183, 183);
+  visibility: visible;
+  border: 2px solid rgb(17, 17, 17);
+  border-radius: 0px;
+  box-shadow: none;
+  z-index: 3;
```

Comment:
can this z-index be removed? it doesn't look like it's having an effect

Reply:
Agreed, I will remove.